### ISSUE TITLE
HPO creates empty label set collection

### ISSUE THREAD

**Issue by kmanpearl** [krishnanlab/obnb#497]
When initializing an [HPO object](https://github.com/krishnanlab/obnb/blob/main/src/obnb/data/annotated_ontology/hpo.py), the label set collection is empty. I suspect this is because HPO uses the mondo terms beginning with `HP:` and something changed in the mondo ontology structure. Eventually we would like to be able to use other organism phenotypes as well as human so instead of using mondo to get HPO terms, it might be best to ingest [phenio](https://github.com/monarch-initiative/phenio) instead, which contains terms from multiple species specific phenotype ontologies. Right now we only need the human (`HP:`) terms within phenio, but down the line we would like to be able to create label sets for the other model organisms that are included as well. If it is easier to ingest these ontologies all individually, then for now just the [HPO](https://hpo.jax.org/) is needed.
```
from obnb.data import DisGeNET, HPO
root = '../data/obnb/FullyRedundant'
lsc = DisGeNET(root=root)
print(len(lsc.sizes)) # 2202

lsc = HPO(root=root)
print(len(lsc.sizes)) # 0
```
